Dockerfile-dind.template

```diff
@@ -53,6 +54,33 @@ RUN set -eux; \
 	; \
 	rm docker.tgz; \
 	\
+	echo '{{
+		{
+			name: "docker",
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: .alpine
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/docker.spdx.json; \
+	echo '{{
+		{
+			name: "docker-compose",
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: .alpine
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/docker-compose.spdx.json; \
```

Comment:
The `dind` image shouldn't need a new `docker-compose` spdx since it is `FROM` the cli image that should already add that.  As for the `docker` one, that should maybe be named `dockerd` since the `cli` image already added a `docker` for the Docker cli binary.